conf/routes

```diff
@@ -61,6 +61,7 @@ GET   /player/myself                   controllers.User.myself
 GET   /player/opponents                controllers.User.opponents
 GET   /player                          controllers.User.list
 GET   /player/top/:nb/:perfKey         controllers.User.topNb(nb: Int, perfKey: String)
+GET   /player/top/:nb/active           controllers.User.topNbActive(nb: Int)
```

Comment:
The line above also matches this route, so it is never selected. Swapping them should work.

Reply:
Thanks, that worked.